src/circular-progress.jsx

```diff
@@ -112,6 +113,8 @@ const CircularProgress = React.createClass({
       AutoPrefix.set(wrapper.style, 'transitionDuration', '10s');
       AutoPrefix.set(wrapper.style, 'transitionTimingFunction', 'linear');
     }, 50);
```

Comment:
Hmmm maybe we should do it all th way. I didn't bother since the timeout is 50 and is so fast relative to 750 or 250. I think the probability of this giving an error is super small. But you're right we should make it deterministic. relying on probabilities are never good :grin: